src/pk.c

```diff
@@ -30,6 +30,14 @@
     #include <wolfssl/wolfcrypt/random.h>
 #endif
 
+#ifdef HAVE_ECC
+    #include <wolfssl/wolfcrypt/ecc.h>
+#endif
+#ifndef WOLFSSL_HAVE_ECC_KEY_GET_PRIV
```

Comment:
This replicated section... is it to support use against older FIPS snapshots that won't have `wc_ecc_key_get_priv` and need the `&` deference?